epyqlib/device.py

```diff
@@ -1140,6 +1147,10 @@ def tab_changed(self, index):
             self.nv_looping_set.start()
             self.nv_tab_looping_set.stop()
 
+        #TODO: Remove this and find a better way to manage when files view should fetch its list
```

Comment:
Not sure. This might end being fine for now. 

One of the areas that I think will need a good amount of polish is figuring out the workflow of how people will use the files tab. For example, are people generally sending and receiving from the inverter when they load the `.epz` file for the inverter?

If so, then can we go ahead and get the inverter serial number as soon as the `.epz` is loaded? If so, then that might be a good time to load the associations for that inverter. 

On the other hand, if users are primarily connecting to the inverter when they're offline (i.e. on a site with no internet) then maybe this won't work.